static/js/directives/repo-view/repo-panel-tags.js

```diff
@@ -67,6 +69,34 @@ angular.module('quay').directive('repoPanelTags', function () {
         });
       };
 
+      var matchCosignSignature = function(tag) {
+        // matches cosign style tags and returns the match
+
+        var cosignNamingPattern = new RegExp('^sha256-[0-9a-f]{64}\.sig$');
+        return tag.match(cosignNamingPattern);
+      }
+
+      var getCosignSignatures = function() {
+        if (!$scope.repositoryTags || !$scope.selectedTags) { return; }
+
+        // Build a list of all digests which are signed by cosign
+        $scope.cosignedManifests = [];
+
+        for (var tag in $scope.repositoryTags) {
+          if (!$scope.repositoryTags.hasOwnProperty(tag)) { continue; }
+
+          var cosignSignatureTag = matchCosignSignature(tag);
+
+          if (cosignSignatureTag) { 
+            signedManifestDigest = cosignSignatureTag[1]; // cosign signature tags contain the signature of the signed manifest
+            $scope.cosignedManifests["sha256:" + signedManifestDigest] = { // map signed manifests to their cosign signature artifact
```

Comment:
`cosignedManifests` seems to be defined as an array on line 83. But you seem to be using it as a dictionary here.